doc/cabal-project.rst

```diff
@@ -194,8 +194,10 @@ Specifying Packages from Remote Version Control Locations
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
```

Comment:
To be uniform with the other paragraph and to cut some fluff, how about:

> Since version 2.4, the source-repository-package stanza allows to specify…